dashboard/lib/api/streamingStats.ts

```diff
@@ -0,0 +1,126 @@
+/*
+ * Copyright 2025 RisingWave Labs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+import { ChannelStatsSnapshot } from "../../pages/fragment_graph"
+import {
+  ChannelDeltaStats,
+  FragmentStats,
+  GetStreamingPrometheusStatsResponse,
+  GetStreamingStatsResponse,
+  RelationStats,
+} from "../../proto/gen/monitor_service"
+import api from "./api"
+
+export type StatsType = "relation" | "fragment"
+
+export interface StreamingStatsCallbacks {
+  setChannelStats: (stats: Map<string, ChannelDeltaStats> | undefined) => void
+  setRelationStats?: (
+    stats: { [key: number]: RelationStats } | undefined
+  ) => void
+  setFragmentStats?: (
+    stats: { [key: number]: FragmentStats } | undefined
+  ) => void
+  toast: (
+    error: any,
+    status?: "info" | "warning" | "success" | "error" | "loading"
+  ) => void
+}
+
+export function createStreamingStatsRefresh(
```

Comment:
Here we unify the refresh logic for both the `fragment` and `relation` graph, they are around the same, besides for updating fragment node vs relation nodes stats.